prometheus/generic/setup-haproxy-exporter/tasks/prereqs.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Ensure epel-release is installed
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - epel-release
+
+- name: Ensure pip is installed
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - python-pip
```

Comment:
Can we combine these two into one task?

Reply:
Nope, python-pip is dependent on epel-release